EventListener/BodyListener.php

```diff
@@ -144,7 +144,7 @@ public function onKernelRequest(GetResponseEvent $event)
      */
     private function isNotAnEmptyDeleteRequestWithNoSetContentType($method, $content, $contentType)
     {
-        return false === ('DELETE' === $method && empty($content) && null === $contentType);
+        return false === ('DELETE' === $method && empty($content) && empty($contentType));
```

Comment:
I would add a test to avoid future regressions. It might not be obvious at a later time why `empty()` is used here.